src/blockchain/ain/jellyfish/utils/raw-tx-utxo.ts

```diff
@@ -0,0 +1,96 @@
+import { Vout, Script } from '@defichain/jellyfish-transaction';
+import BigNumber from 'bignumber.js';
+import { Config } from 'src/config/config';
+import { UtxoConfig } from '../domain/entities/utxo-config';
+import { UtxoSizePriority } from '../domain/enums';
+import { RawTxDto } from '../dto/raw-tx.dto';
+import { RawTxBase } from './raw-tx-base';
+import { RawTxUtil } from './raw-tx-util';
+
+export class RawTxUtxo extends RawTxBase {
+  // SEND UTXOS //
+  async sendFromTo(from: string, to: string, amount: BigNumber): Promise<RawTxDto> {
+    return this.handle(() =>
+      this.send(from, to, amount, true, { useFeeBuffer: false, sizePriority: UtxoSizePriority.SMALL }),
+    );
+  }
+
+  async sendFromLiq(to: string, amount: BigNumber, sizePriority: UtxoSizePriority): Promise<RawTxDto> {
+    return this.handle(() =>
+      this.send(Config.staking.liquidity.address, to, amount, true, { useFeeBuffer: true, sizePriority }),
+    );
+  }
+
+  async sendToLiq(from: string, amount: BigNumber): Promise<RawTxDto> {
+    return this.handle(() =>
+      this.send(from, Config.staking.liquidity.address, amount, false, {
+        useFeeBuffer: false,
+        sizePriority: UtxoSizePriority.SMALL,
+      }),
+    );
+  }
+
+  private async send(
+    from: string,
+    to: string,
+    amount: BigNumber,
+    useChangeOutput: boolean,
+    config: UtxoConfig,
+  ): Promise<RawTxDto> {
+    const [fromScript, fromPubKeyHash] = RawTxUtil.parseAddress(from);
+    const [toScript] = RawTxUtil.parseAddress(to);
+
+    const utxo = useChangeOutput
+      ? await this.utxoProvider.provideUntilAmount(from, amount, config)
+      : await this.utxoProvider.provideExactAmount(from, amount);
+
+    const vins = RawTxUtil.createVins(utxo.prevouts);
+    const vouts = [RawTxUtil.createVoutReturn(toScript, amount)];
+    if (useChangeOutput) {
+      const change = RawTxUtil.createVoutReturn(fromScript, utxo.total?.minus(amount));
+      vouts.push(change);
+    }
+    const fromWitness = RawTxUtil.createWitness([RawTxUtil.createWitnessScript(fromPubKeyHash)]);
+    const witnesses = new Array(vins.length).fill(fromWitness);
+
+    return RawTxUtil.generateTxAndCalcFee(utxo, vins, vouts, witnesses);
+  }
+
+  // MANAGE UTXOS //
+  async split(address: string, split: number): Promise<RawTxDto> {
+    return this.handle(() => this.management(address, split, { sizePriority: UtxoSizePriority.BIG }));
+  }
+
+  async merge(address: string, merge: number): Promise<RawTxDto> {
+    return this.handle(() => this.management(address, merge, { sizePriority: UtxoSizePriority.SMALL }));
+  }
+
+  private async management(address: string, numberOf: number, config: UtxoConfig): Promise<RawTxDto> {
+    const numberOfInputs = config.sizePriority === UtxoSizePriority.SMALL ? numberOf : 1;
+    const numberOfOutputs = config.sizePriority === UtxoSizePriority.BIG ? numberOf : 1;
```

Comment:
I think we already discussed this, small detail 😉 this is IMO some Logic that belongs into the calling methods, because only they know, if they want to have many inputs or outputs. And it makes the code even simpler.
I would use two parameters `management(address: string, inputCount: number, outputCount: number, config: UtxoConfig)`